### DISCUSSION TITLE
Is it possible to add the number of "SLOCs" per game under the heading of the audit report

### DISCUSSION THREAD

**Discussion by IAm0xAxe** [solodit/feedback#3] [New Feature Requests]
Because of the addition of "SLOCs" is more conducive to newcomers to filter for their own matches (PS: small matches with small code base). It would also be great if the search box could be searched by the range of "SLOCs"! Combined with the contest type search, this will help auditors of all levels to find the right contest for them!

For example:
![1](https://github.com/solodit/feedback/assets/33568293/77b6fcd5-9244-4d56-90ea-01739f24a59c)

**Comment by IAm0xAxe** [comment]
Hey man. I mentioned in solodit's Github a while ago "Is it possible to add the number of SLOC per game in the title of the audit report", I have now compiled the C4 Sloc, but the number is too large for me to upload via Github, you can access it via Notion's link - https://lofty-trumpet-86c.notion.site/C4-Sloc-9bd5b1246db64191a00e4551c36654eb . Some places are 0 or empty, indicating a private audit competition or I am not sure how many Sloc should be.  

![1](https://github.com/solodit/feedback/assets/33568293/6b87cf44-d98e-4ac1-bc17-be5aea142569)

**Reply by hans-solodit** [reply]
Thanks, we will work on this soon.

**Reply by IAm0xAxe** [reply]
I've just finished Sherlock's as well, and I've added it to https://www.notion.so/Sherlock-Sloc-ffdadbe9c1f6433ea83b22927459bf93
![image](https://github.com/solodit/feedback/assets/33568293/df3d03bb-f521-4100-9fca-f1ba23dd9723)

**Comment by IAm0xAxe** [comment]
Hey,I'm still looking forward to it. >_<

**Reply by hans-cyfrin** [reply]
Hi, @IAm0xAxe.
This feature is tricky because we also support traditional audit firms and the SLOC information is only sometimes on the report.
Furthermore, we don't always have access to their codebase.

**Reply by IAm0xAxe** [reply]
It's okay, bro @hans-cyfrin . 
That part I can refine myself using the form.
Thank you.

**Comment by hans-cyfrin** [comment]
Reconsidering this request, it would not be that beneficial to have nSLOC for the findings. (while I admit it is important for on-going contests to help auditors choose the target to work).
